sql/core/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.3.0</version>
+      <version>2.4.4</version>
```

Comment:
I see this has already been closed, but this version tag should just be removed instead, so that it's inherited from the parent pom.